Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8192,3 +8192,30 @@ def get_tenant_account_name():
         account_name = "acc_{}".format(tenant_name) if tenant_name != "" else ""
 
     return account_name
+
+def indicators_value_to_clickable(indicators) -> Dict[str, str]:
```

Comment:
This is not supported in CSP.
```suggestion
def indicators_value_to_clickable(indicators):
```